src/clj/puppetlabs/services/jruby/jruby_puppet_core.clj

```diff
@@ -4,27 +4,31 @@
             [schema.core :as schema]
             [puppetlabs.kitchensink.classpath :as ks-classpath]
             [puppetlabs.kitchensink.core :as ks]
-            [puppetlabs.services.jruby.jruby-puppet-schemas :as jruby-schemas]
+            [puppetlabs.services.jruby.jruby-puppet-schemas :as jruby-puppet-schemas]
+            [puppetlabs.services.jruby-pool-manager.jruby-schemas :as jruby-schemas]
+            [puppetlabs.services.jruby-pool-manager.jruby-core :as jruby-core]
+            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-puppet-core]
```

Comment:
When running `user-repl/reset` in a REPL on this branch, I got the following error:

~~~
Exception Circular dependency between puppetlabs.services.jruby.jruby-puppet-core and puppetlabs.services.jruby.jruby-puppet-core  clojure.tools.namespace.dependency.MapDependencyGraph (dependency.clj:89)
~~~

I found that if I removed this line and changed the references to it to just call the related functions without an alias that the error no longer occurred.  This seems unnecessary in any event, so probably worth removing.